proposals/lip-0054.md

```diff
@@ -221,10 +221,7 @@ def verify(trs: Transaction) -> None:
     if recoveryModule does not have a recover function:
         raise Exception("Module is not recoverable.")
 
-    terminatedStateRoot = terminatedStateAccount(trsParams.chainID).stateRoot
-
     queryKeys = []
-    storeQueries = []
 
     # Calculate store prefix from the module name according to LIP 0040.
     storePrefix = module_name_to_store_prefix(trsParams.module)
```

Comment:
I think now we can remove this (and update the queryKey) since we do not check the inclusion proof here

Reply:
for aesthetics reasons you may want to keep this tho, up to you

Reply:
But then you need to initialiaze queryKey as an empty list, which would be almost the same, so I would not bother modifying this.

Reply:
I agree that it is not needed here. So, I removed and also renamed `queryKeys` to `keys`.
@gkoumout I'm not sure what exactly you mean with "then you need to initialiaze queryKey as an empty list" because this already the case now.

Reply:
Ok this comment is outdated. I meant that if if `queryKeys` disappeared, then we would have to initialize the variable `queryKey` which was used inside the for-loop. 

Anyways, does not matter anymore. Current status looks good.